backend/coins/btc/db/headersdb/headersdb.go

```diff
@@ -21,23 +21,58 @@ import (
 	"github.com/btcsuite/btcd/wire"
 	"github.com/digitalbitbox/bitbox-wallet-app/util/errp"
 	"github.com/digitalbitbox/bitbox-wallet-app/util/locker"
+	"github.com/sirupsen/logrus"
 )
 
 const headerSize = 80
 
 // DB is a bbolt key/value database.
```

Comment:
It is not a bbolt DB anymore, can you update this?

Reply:
Done :+1: